packages/opine/opine.0.1.3/opam

```diff
@@ -0,0 +1,52 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Python AST unparse implementation in OCaml"
+description:
+  "Opine is an OCaml library that unparses the python AST back to python code."
+maintainer: ["arulselvan1234@gmail.com"]
+authors: ["Arulselvan Madhavan"]
+license: "MIT"
+tags: ["topics" "ocaml" "python" "ast"]
+homepage: "https://github.com/ArulselvanMadhavan/opine"
+doc: "https://url/to/documentation"
```

Comment:
If you do not have any pre-generated documentation somewhere the `doc` field should be omited
```suggestion
```